root/etc/nginx/nginx.conf

```diff
@@ -48,6 +49,24 @@ http {
     # for more information.
     include /etc/nginx/conf.d/*.conf;
 
+    proxy_cache_path  /etc/nginx/openshiftio_cache levels=1:2 keys_zone=openshiftio_cache:10m max_size=10g inactive=60m use_temp_path=off;
```

Comment:
note the path setup in the dockerfile isnt openshiftio_cache but just cache

Reply:
https://github.com/fabric8-ui/fabric8-ui-openshift-nginx/pull/4/files#diff-3254677a7917c6c01f55212f86c57fbfR18 as ref